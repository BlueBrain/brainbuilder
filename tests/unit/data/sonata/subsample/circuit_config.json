{
  "version": "2",
  "manifest": {
    "$BASE_DIR": "."
  },
  "networks": {
    "nodes": [
      {
        "nodes_file": "$BASE_DIR/nodes.h5",
        "populations": {
          "default": {
            "type": "biophysical",
            "morphologies_dir": "$BASE_DIR/swc",
            "biophysical_neuron_models_dir": "/emodel/dir"
          }
        }
      }
    ],
    "edges": [
      {
        "edges_file": "$BASE_DIR/edges.h5",
        "populations": {
          "default": {
          }
        }
      }
    ]
  }
}
