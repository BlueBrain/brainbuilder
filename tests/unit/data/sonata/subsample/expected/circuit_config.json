{
  "version": "2",
  "manifest": {
    "$BASE_DIR": "."
  },
  "networks": {
    "nodes": [
      {
        "nodes_file": "$BASE_DIR/networks/nodes/default/nodes.h5",
        "populations": {
          "default": {}
        }
      }
    ],
    "edges": [
      {
        "edges_file": "$BASE_DIR/networks/edges/default/edges.h5",
        "populations": {
          "default": {}
        }
      }
    ]
  }
}